app/src/main/java/org/helfoome/presentation/MainActivity.kt

```diff
@@ -1,12 +1,93 @@
 package org.helfoome.presentation
 
+import android.content.Intent
+import android.graphics.Paint
+import android.net.Uri
 import android.os.Bundle
+import android.view.View
+import androidx.activity.viewModels
+import androidx.core.widget.NestedScrollView
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.tabs.TabLayoutMediator
+import dagger.hilt.android.AndroidEntryPoint
 import org.helfoome.R
 import org.helfoome.databinding.ActivityMainBinding
+import org.helfoome.presentation.restaurant.RestaurantTabAdapter
 import org.helfoome.util.binding.BindingActivity
 
+@AndroidEntryPoint
 class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main) {
+    private val viewModel: MainViewModel by viewModels()
+    private lateinit var behavior: BottomSheetBehavior<NestedScrollView>
+    private val restaurantDetailAdapter = RestaurantTabAdapter(this)
+    private val tabTitles = arrayOf(
+        R.string.restaurant_detail_tab_item_menu,
+        R.string.restaurant_detail_tab_item_how_to_eat_out,
+        R.string.restaurant_detail_tab_item_review,
+    )
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this@MainActivity
+
+        initView()
+        initListeners()
+        initObservers()
+    }
+
+    override fun onStart() {
+        super.onStart()
+        behavior.addBottomSheetCallback(bottomSheetCallback)
+    }
+
+    private fun initView() {
+        behavior = BottomSheetBehavior.from(binding.svBottomSheet)
+
+        with(binding.layoutRestaurantDialog) {
+            vpRestaurantDetail.adapter = restaurantDetailAdapter
+            TabLayoutMediator(layoutRestaurantTabMenu, vpRestaurantDetail) { tab, position ->
+                tab.text = getString(tabTitles[position])
```

Comment:
저 위에 tabTitles라는 리스트를 삭제하고, strings.xml에서 **<string-array>** 활용해서,

**strings.xml**
```
    <string-array name="tab_titles">
        <item>메뉴</item>
        <item>외식대처법</item>
        <item>리뷰</item>
    </string-array>
```

이렇게 선언하고,

```suggestion
                tab.text = stringListFrom(R.array.tab_titles)[position]
```

이렇게 표현해도 간결할 것 같아요😊😊

Reply:
오호라,, 굳이 tabTitles라는 리스트를 따로 만들지 않아도 되겠군요!!! 역쉬,,, 최고십니다 ❤️